CHANGELOG.md

```diff
@@ -4,6 +4,63 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.57.0]
+
+### Major/Breaking Changes
```

Comment:
Can we separate the "Major" and "Breaking" changes? We haven't been consistent in previous releases (some mixed them, and some combined them). But it's more convenient for the reader to have them separated.

Reply:
Agreed and done.